src/main/java/org/folio/rest/impl/PatronBlockLimitsAPI.java

```diff
@@ -113,13 +114,8 @@ private Errors validateRangeForDoubleValueType(Double limit) {
     if (limit >= MIN_DOUBLE_LIMIT && limit <= MAX_DOUBLE_LIMIT) {
       return null;
     }
-    if (limit == 0.00) {
-      return createValidationErrorMessage(VALUE_FIELD, limit.toString(),
-        "A maximum balance of 0 will result in all patrons in this group being blocked; " +
-          "to skip this limit, leave value set to blank");
-    }
     return createValidationErrorMessage(VALUE_FIELD, limit.toString(),
-      "Must be blank or a number from " + MIN_DOUBLE_LIMIT + " to " + MAX_DOUBLE_LIMIT);
+      "Must be blank or a number from " + new MonetaryValue(MIN_DOUBLE_LIMIT) + " to " + MAX_DOUBLE_LIMIT);
```

Comment:
If we're using `MonetaryValue` already, shouldn't we also use it for `MAX_DOUBLE_LIMIT`?